Update/omot_005.txt

```diff
@@ -1528,7 +1528,7 @@ void main()
 //r徒手空拳の今の私では、良くて相打ち。いずれにしても、自分が殺される日につながる地獄の時計の針を早めるだけだった。
 	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
 	OutputLine(NULL, "徒手空拳の今の私では、良くて相打ち。いずれにしても、自分が殺される日につながる地獄の時計の針を早めるだけだった。",
-			NULL, "Since I'm unarmed, she'd be a fair match for me. At any rate, I'd just be speeding up the hands on the clock of Hell approaching the day of my death.", Line_Normal);
+			NULL, "Since I'm unarmed, I could make us take each other down at best. At any rate, I'd just be speeding up the hands on the clock of Hell approaching the day of my death.", Line_Normal);
```

Comment:
```suggestion
			NULL, "Since I'm unarmed, at best we'd take each other out. At any rate, I'd just be speeding up the hands on the clock of Hell approaching the day of my death.", Line_Normal);
```